src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql

```diff
@@ -179,13 +179,40 @@ VALUES
+		,'[SigninId],[SigninType],[CreatedBy]',0,1
+		,'select [Id],[Email],[GivenNames],[FamilyName],[SigninId],[SigninType],[ApplyOrganisationID],[ContactDetails],[Status],[IsApproved],[CreatedAt],case when TRY_CONVERT(UNIQUEIDENTIFIER,[CreatedBy]) IS NOT NULL Then [CreatedBy] Else  STUFF([CreatedBy],2,len([CreatedBy])-2,REPLICATE(''''*'''',len([CreatedBy])-2)) End As [CreatedBy],[UpdatedAt],[DeletedAt]  from [dbo].[Contacts]','RP_Contacts')
+,('Apply','FinancialData','dbo','[Id],[ApplicationId]',''
+		,'[TurnOver],[Depreciation],[ProfitLoss],[Dividends],[IntangibleAssets],[Assets],[Liabilities],[ShareholderFunds],[Borrowings],[AccountingReferenceDate],[AccountingPeriod],[AverageNumberofFTEEmployees]',0,1
+		,'select [Id],[ApplicationId],[TurnOver],[Depreciation],[ProfitLoss],[Dividends],[IntangibleAssets],[Assets],[Liabilities],[ShareholderFunds],[Borrowings],[AccountingReferenceDate],[AccountingPeriod],[AverageNumberofFTEEmployees]  from [dbo].[FinancialData]','RP_FinancialData')
+,('Apply','GatewayAnswer','dbo','[Id],[ApplicationId],[PageId],[UpdatedAt]',''
+		,'[Status],[Comments],[ClarificationComments],[ClarificationDate],[ClarificationBy],[ClarificationAnswer],[GatewayPageData]',0,1
+		,'select [Id],[ApplicationId],[PageId],[Status],[Comments],[ClarificationComments],[ClarificationDate],STUFF([ClarificationBy],2,len([ClarificationBy])-2,REPLICATE(''''*'''',len([ClarificationBy])-2)) As [ClarificationBy],[ClarificationAnswer],[GatewayPageData],[UpdatedAt] from [dbo].[GatewayAnswer]','RP_GatewayAnswer')
+,('Apply','ModeratorPageReviewOutcome','dbo','[Id],[ApplicationId],[SequenceNumber],[SectionNumber],[PageId],[ClarificationUpdatedAt],[CreatedAt],[UpdatedAt]',''
+		,'[ModeratorUserId],[ModeratorReviewStatus],[ModeratorReviewComment],[ClarificationUserId],[ClarificationStatus],[ClarificationComment],[ClarificationResponse],[ClarificationFile],[CreatedBy],[ModeratorUserName],[ClarificationUserName]',0,1
+		,'Select [Id],[ApplicationId],[SequenceNumber],[SectionNumber],[PageId],STUFF([ModeratorUserId],2,len([ModeratorUserId])-2,REPLICATE(''''*'''',len([ModeratorUserId])-2)) As [ModeratorUserId],STUFF([ModeratorUserName],2,len([ModeratorUserName])-2,REPLICATE(''''*'''',len([ModeratorUserName])-2)) As [ModeratorUserName],[ModeratorReviewStatus],[ModeratorReviewComment],STUFF([ClarificationUserId],2,len([ClarificationUserId])-2,REPLICATE(''''*'''',len([ClarificationUserId])-2)) As [ClarificationUserId],STUFF([ClarificationUserName],2,len([ClarificationUserName])-2,REPLICATE(''''*'''',len([ClarificationUserName])-2)) As [ClarificationUserName],[ClarificationStatus],[ClarificationComment],[ClarificationResponse],[ClarificationFile],[ClarificationUpdatedAt],[CreatedAt],case when TRY_CONVERT(UNIQUEIDENTIFIER,[CreatedBy]) IS NOT NULL Then [CreatedBy] Else  STUFF([CreatedBy],2,len([CreatedBy])-2,REPLICATE(''''*'''',len([CreatedBy])-2)) End As [CreatedBy],[UpdatedAt] from [dbo].[ModeratorPageReviewOutcome]','RP_ModeratorPageReviewOutcome')
```

Comment:
Might be good to add the same Checks to see if it is guid for all User Id Columns. In this case ModeratorUserId

Reply:
Waiting for Prod Data to arrive to see how the User Id looks like with Live Data. Until then it's agreed to keep it masked to be safe. Will need revisiting once we have Live Data.